examples/head_or_tails.py

```diff
@@ -0,0 +1,16 @@
+import random
+
+from nextcord.ext import commands
+
+bot = commands.Bot(command_prefix="$")
+
+@bot.command()
+async def headortails(ctx, answer=None:
```

Comment:
You are missing a ) here

Reply:
I am so sorry If i make so many mistakes :(